pyproject.toml

```diff
@@ -17,7 +17,6 @@ keywords = ["Momento", "caching", "key-value store", "serverless"]
 classifiers = [
   "Development Status :: 3 - Alpha",
   "Operating System :: OS Independent",
-  "Framework :: AsnycIO",
```

Comment:
should this just be `AsyncIO`?  it's spelled wrong

Reply:
can try it good catch 🦅 👁️